### ISSUE TITLE
New project yields blank API page

### ISSUE THREAD

**Issue by oddjobz** [rtfd/readthedocs.org#2785]
## Details

* Project URL: https://github.com/oddjobz/pymamba
* Build URL (if applicable): http://pymamba.readthedocs.io/en/latest/#
* Read the Docs username (if applicable): oddjobz

## Expected Result

I'm expecting to see a html page with API details as I do locally.

## Actual Result

I see a paragraph header for the module, but nothing else.
The page is generated by Sphinx on my machine quite happily, and the html is there in the repo.
Works for me using Python2 and Python3.

**Comment by humitos** [comment]
Hello! The page you linked under Read The Docs doesn't work. Can you provide the project URL at RTD and the build link? Thanks!

**Comment by oddjobz** [comment]
Hi, sorry but it was being found by Google as blank documentation so I've had to delete the account. I'm currently pushing it out to a webhost here; https://pymamba.linux.co.uk/

The project source is here;
https://github.com/oddjobz/pymamba

The docs stuff that's currently building and working for the above link is in the docs folder.

**Comment by humitos** [comment]
Thanks. I'm confused, what's the issue you want to report on RTD?

**Comment by oddjobz** [comment]
The issue was that 11 days ago I tried to publish some documentation, and it didn't work, leaving me with a public profile and no documentation. After waiting a little while and getting no response, I decided to remove the profile so people would not find the page on google and have it show them no documentation. I should have closed this issue but forgot, my apologies. With regards to the links I supplied, I was merely documenting that the processes involved seem to work elsewhere.
